docs/source/contribution/technical_steering_committee.md

```diff
@@ -34,34 +37,83 @@ In this section, we describe:
 Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
 working with existing maintainers for a period of time.
 
-We look for commitment markers with the following:
+We look for commitment markers who can do the following:
 
-- Being able to write solid code and collaborate with the team and community
-- Understanding the project's code base and internals
-- Doing pull requests from our backlog or roadmap; maintainers need to work towards a common goal
-- Learning how the team works, including processes for testing, quality standards and code review
-- Evidence of already having started pull requests and code reviews under the guidance of maintainers; including asking
+- Write high-quality code and collaborate with the team and community
+- Understand the project's code base and internals
+- Make pull requests from our backlog or roadmap; maintainers need to work towards a common goal
+- Learn how the team works, including processes for testing, quality standards and code review
+- Show evidence of already having started pull requests and code reviews under the guidance of maintainers; including asking
   for help where needed
-- Showing excitement about the future of Kedro
-- Building a collaborative relationship with the existing team
+- Show excitement about the future of Kedro
+- Build a collaborative relationship with the existing team
+
+## Kedro maintainers
+
+Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) and [Nikolaos Tsaousis](https://github.com/tsanikgr) at QuantumBlack to solve challenges they faced in their project work. Their work was later turned into an internal product by [Peteris Erins](https://github.com/Pet3ris), [Ivan Danov](https://github.com/idanov), [Nikolaos Kaltsas](https://github.com/nikos-kal), [Meisam Emamjome](https://github.com/misamae) and [Nikolaos Tsaousis](https://github.com/tsanikgr).
+
+
+Currently, the core Kedro team consists of:
```

Comment:
Not for this PR, but maybe it would be cool if we could show pictures too, like https://github.com/readthedocs/readthedocs.org/pull/8441 (not live anymore)